### PR TITLE
add module rules_tf

### PR DESCRIPTION
**Pull request by yanndegat**
this PR add module https://github.com/yanndegat/rules_tf to the BCR

### CODE REVIEW

File: modules/rules_tf/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "homepage": "https://github.com/yanndegat/rules_tf/",
+    "maintainers": [],
```
Comment (fmeum) [inline modules/rules_tf/metadata.json:3]: Could you add yourself as a maintainer here?

**Comment by yanndegat** [comment]
hi @fmeum 

do you know why i'm hitting this issue : https://buildkite.com/bazel/bcr-presubmit/builds/2936#018cd966-3ede-4bcc-8826-d4cac252a986

thanks a lot

**Comment by fmeum** [comment]
Hmm, I don't. Cc @meteorcloudy

**Comment by yanndegat** [comment]
> This is because the MODULE.bazel file in the tar file has no write permission:
> 
> ```
> $ ll
> total 48
> drwxr-xr-x   8 pcloudy  primarygroup    256 Jan  5 15:15 ./
> drwxr-xr-x  14 pcloudy  primarygroup    448 Jan  5 15:15 ../
> -r--r--r--   1 pcloudy  primarygroup    686 Jan  1  2000 BUILD.bazel
> -r--r--r--   1 pcloudy  primarygroup  11358 Jan  1  2000 LICENSE
> -r--r--r--   1 pcloudy  primarygroup   1840 Jan  1  2000 MODULE.bazel
> -r--r--r--   1 pcloudy  primarygroup      0 Jan  1  2000 WORKSPACE
> drwxr-xr-x  10 pcloudy  primarygroup    320 Jan  1  2000 tf/
> -r--r--r--   1 pcloudy  primarygroup    652 Jan  1  2000 version.bzl
> ```
> 
> Fixing the CI script at [bazelbuild/continuous-integration#1844](https://github.com/bazelbuild/continuous-integration/pull/1844)

hi @meteorcloudy 

thanks a lot for having fixed this.

i'm sorry but it seems i'm hitting another issue:

```
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/buildkite-agent/builds/bk-docker-j2dn/bazel-org-repo-root/output/tests/bcr/.bazelrc'
```

is this something i should include in my package ?


thanks a lot

**Comment by meteorcloudy** [comment]
It looks like the test module directory [`tests/bcr`](https://github.com/bazelbuild/bazel-central-registry/pull/1265/files#diff-55e55e5f82d58bd13c60a96ac694288b3c60701e69c6a6c283a41fb283fc9444R17) doesn't exist in the source archive.

**Comment by yanndegat** [comment]
> It looks like the test module directory [`tests/bcr`](https://github.com/bazelbuild/bazel-central-registry/pull/1265/files#diff-55e55e5f82d58bd13c60a96ac694288b3c60701e69c6a6c283a41fb283fc9444R17) doesn't exist in the source archive.

oh correct. i didn't think i had to include it in the archive

will fix

thanks a lot

**Comment by yanndegat** [comment]
hi @fmeum 

i finally managed to make it green. is there anything i should do? 

thanks a lot.

best regards

**Review by Wyverald — APPROVED** [review]

**Pull request merged**